addons/cargo/functions/fnc_startUnload.sqf

```diff
@@ -26,7 +26,7 @@ if (GVAR(interactionParadrop)) exitWith {
     // Close the cargo menu
     closeDialog 0;
 
-    private _duration = GVAR(paradropTimeCoefficent) * (_item call FUNC(getSizeItem));
+    private _duration = [_item] call FUNC(getDelayItem);
```

Comment:
Not the same coefficient is applied here, its `GVAR(paradropTimeCoefficent)` instead of `GVAR(loadTimeCoefficient)`.

Reply:
three solutions i can think of, none ideal. any comments on any?

1) in each call site, perform the new check instead of using this function
2) pass in as a parameter, `_delayCoefficient`. This feels a bit hacky, but is probably the best
3) only return the item size, since the size == the time with a coefficient of one, this function is still accurate. At each call site, scale by the coefficient. I can see merits to this solution

Reply:
maybe variation on 2
pass in `_isParadop` true/false